db/schema/usersSchema.js

```diff
@@ -1,10 +1,17 @@
 module.exports = function(knex) {
   return knex.schema.createTableIfNotExists('users', function(table) {
     table.increments('u_id').primary();
-    table.string('username');
+    table.string('username').unique();
```

Comment:
This shouldn't be unique. UserID (and FacebookID) should be uniquely constrained